node/src/Pos/Explorer/Txp/Local.hs

```diff
@@ -172,10 +177,10 @@ eTxProcessTransactionNoLock itw@(txId, txAux) = reportTipMismatch $ runExceptT $
         -> HeaderHash
         -> (TxId, TxAux)
         -> Maybe Timestamp
-        -> ETxpLocalDataPure
-        -> (Either ToilVerFailure (), ETxpLocalDataPure)
-    processTxDo curEpoch ctx@EProcessTxContext {..} tipBefore tx mTxTimestamp txld@(uv, mp, undo, tip, extra)
-        | tipBefore /= tip = (Left $ ToilTipsMismatch tipBefore tip, txld)
+        -> (UtxoModifier, MemPool, UndoMap, HeaderHash, ExplorerExtra)
```

Comment:
Interesting "unrolling" -- what was the problem with `ETxpLocalDataPure`?

Reply:
Nothing, really. I removed it since I removed `GenericTxpLocalDataPure`, but there's no reason I couldn't just unroll the alias. Might actually do that...